Gaijinizer patch/Map447.txt

```diff
@@ -3,446 +3,570 @@
-
+> CONTEXT: Map447/events/1/pages/0/119/Dialogue/enemy15
+Ah! Don't tell me you're here on some test of
+courage or something? This shrine does have quite
+an atmosphere at night. Did you get attacked by
+that other girl when you came to have a look?#Ron
 > END STRING
 
 > BEGIN STRING
 い、いや………
 
 そういうのじゃないんだけど………その……
-> CONTEXT: Map447/events/1/pages/0/126/Dialogue/kazuya-a < UNTRANSLATED
+> CONTEXT: Map447/events/1/pages/0/126/Dialogue/kazuya-a
+N-No...
 
+It's not any of those things... umm...#Ron
 > END STRING
 
 > BEGIN STRING
 （ち、近っ…！おっぱいめっちゃ揺れてる……
 
 　わ、わざとか…？）
-> CONTEXT: Map447/events/1/pages/0/130/Dialogue/kazuya-a < UNTRANSLATED
-
+> CONTEXT: Map447/events/1/pages/0/130/Dialogue/kazuya-a
+(S-She's super close! And her boobs are swaying so
+ much... Is she doing it on purpose?)#Ron
 > END STRING
 
 > BEGIN STRING
 （だ、ダメだ動揺しちゃ……
 
 　さっき『にじみ出てる』とか言われたばかりじゃないか…）
-> CONTEXT: Map447/events/1/pages/0/134/Dialogue/kazuya-a < UNTRANSLATED
+> CONTEXT: Map447/events/1/pages/0/134/Dialogue/kazuya-a
+(N-No! I can't let myself get worked up.
 
+ Not after I was just told how much I was
+ "oozing out" or whatever.)#Ron
 > END STRING
 
 > BEGIN STRING
 それにしても……
 
 さっきの子、なんだか不思議な感じだったね。
 悪魔……とは雰囲気が違う感じ
-> CONTEXT: Map447/events/1/pages/0/142/Dialogue/enemy15 < UNTRANSLATED
+> CONTEXT: Map447/events/1/pages/0/142/Dialogue/enemy15
+Still though...
 
+I could feel something off about that girl from
+before. Different from the aura demons have.#Ron
 > END STRING
 
 > BEGIN STRING
 この結界、あの子が作ったものだと思ってたけど
 消えないところをみると、そうじゃないみたいだし……
 
 この結界って、一体何なんだろう？
-> CONTEXT: Map447/events/1/pages/0/147/Dialogue/enemy15 < UNTRANSLATED
-
+> CONTEXT: Map447/events/1/pages/0/147/Dialogue/enemy15
+I thought that this barrier over us might have
+been her doing, but seeing as how it's still up
+even after she's gone, I suppose it wasn't. Just
+what on earth is it?#Ron
 > END STRING
 
 > BEGIN STRING
 あなたも……魔力は感じるけど、彼女と同じなのかな？
 
 『実体化』しているっていうより……
 『出てきている』って感じ………
-> CONTEXT: Map447/events/1/pages/0/155/Dialogue/enemy15 < UNTRANSLATED
-
+> CONTEXT: Map447/events/1/pages/0/155/Dialogue/enemy15
+And you... I can sense magic power within you,
+but it feels similar to hers. It feels less like
+you've materialized, and more like you've
+"appeared".#Ron
 > END STRING
 
 > BEGIN STRING
 さっきは私のことを知っているみたいな話し方してたけど
 
 もしかして、あなたが私をここに呼んだの？
-> CONTEXT: Map447/events/1/pages/0/160/Dialogue/enemy15 < UNTRANSLATED
-
+> CONTEXT: Map447/events/1/pages/0/160/Dialogue/enemy15
+And you talked about me before like you knew who
+I was. Don't tell me, are you the one who
+summoned me here?#Ron
 > END STRING
 
 > BEGIN STRING
 …………察しがいいね。
 
 ボクや彼女みたいなのは『魔物』……
 さしずめ\\C[2]『モンスター娘』\\C[0]って呼んでくれればいいと思うよ。
-> CONTEXT: Map447/events/1/pages/0/167/Dialogue/sirousagi_fc1 < UNTRANSLATED
+> CONTEXT: Map447/events/1/pages/0/167/Dialogue/sirousagi_fc1
+Bingo.
 
+That girl and I are "monsters"... for now you can
+just call us \\C[2]monmusumes\\C[0].#Ron
```

Comment:
Wouldn't there be no s for a plural here? "monmusume" itself is a bit of a weird mix between abbreviation and not as well.

I think this might be better localized, like...
> That girl and I are "monster girls"... for now you can
> just call us \\C[2]monsters\\C[0].